Formula/kcov.rb

```diff
@@ -0,0 +1,37 @@
+class Kcov < Formula
+  desc "Code coverage tool for compiled programs, Python and Bash"
+  homepage "https://simonkagstrom.github.com/kcov/index.html"
+  url "https://github.com/SimonKagstrom/kcov/archive/v35.tar.gz"
+  sha256 "74c172dae2ac2866e0adc91d3fd80276e5acb970d11ac71679a0f7336897a476"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on :xcode # Needed for LLDB. See MachO work below.
+
+  def install
+    system "cmake", "-G", "Unix Makefiles", *std_cmake_args
+    system "make"
+
+    # As per notes in https://github.com/SimonKagstrom/kcov/issues/166
```

Comment:
We'll need to figure out some upstream patch or different build-system option to use to avoid this; we don't want to carry this indefinitely.

Reply:
The upstream bug has been closed, but it's been closed by doing what I was doing here inside cmake (see https://github.com/SimonKagstrom/kcov/commit/6391c6f68ffd4aa166aa755fc9ddf5dfd58dbe9d)

Reply:
Can you apply that as a patch to the formula?